apps/blink/setup.conf

```diff
@@ -0,0 +1 @@
+CONFIG_BT_DEVICE_NAME="KNoT Blink"
```

Comment:
Use `git mv` to rename the file. This commit shows the previous file deleted and the new one added.
This way we keep the history of the file.

Reply:
Do the same in button sample

Reply:
It didn't work